src/eos/radiationProperties/sootMeanAbsorption.cpp

```diff
@@ -11,7 +11,7 @@ PetscErrorCode ablate::eos::radiationProperties::SootMeanAbsorption::SootFunctio
 
     ierr = functionContext->temperatureFunction.function(conserved, &temperature, functionContext->temperatureFunction.context.get());  //!< Get the temperature value at this location
     CHKERRQ(ierr);
-    ierr = functionContext->densityFunction.function(conserved, &density, functionContext->densityFunction.context.get());  //!< Get the density value at this location
+    ierr = functionContext->densityFunction.function(conserved, temperature, &density, functionContext->densityFunction.context.get());  //!< Get the density value at this location
```

Comment:
Using the temperature based density function speeds the absorptivity up also since we already called for the temperature.